src/model/documentselection.js

```diff
@@ -707,9 +707,15 @@ function clearAttributesStoredInElement( changes, batch ) {
 		return;
 	}
 
-	const storedAttributes = Array.from( changeParent.getAttributeKeys() ).filter( key => key.startsWith( storePrefix ) );
+	const anyStoredAttributes = Array.from( changeParent.getAttributeKeys() ).some( key => key.startsWith( storePrefix ) );
```

Comment:
Maybe I am not right, but...

we need to "refresh" `storedAttributes` array anyway in the `enqueueChanges` because something might have changed.

But, if that's true, we should not have `if( anyStoredAttributes )` because there might be no stored attributes ATM but, at least theoretically, may be some later?

Argh, this `enqueueChanges` is too damn difficult!